config/routes.rb

```diff
@@ -140,12 +137,12 @@
       patch :delete_account, on: :member
     end
 
-    get 'users/:id/stories' => 'users#stories'
-    get 'users/:id/stories/:product_id' => 'users#stories'
-    get '/users/:id/karma' => 'users#karma', :as => :user_karma
-    get '/users/:id/assets' => 'users#assets', :as => :user_assets
-    post '/users/:id/dismiss_welcome_banner' => 'users#dismiss_welcome_banner', :as => :dismiss_welcome_banner
-    post '/users/:id/dismiss_showcase_banner' => 'users#dismiss_showcase_banner', :as => :dismiss_showcase_banner
+    get 'users/:id/stories', to: 'users#stories'
+    get 'users/:id/stories/:product_id', to: 'users#stories'
+    get '/users/:id/karma',  to: 'users#karma',  as: :user_karma
+    get '/users/:id/assets', to: 'users#assets', as: :user_assets
+    post '/users/:id/dismiss_welcome_banner',  to: 'users#dismiss_welcome_banner', as: :dismiss_welcome_banner
```

Comment:
Line is too long. [110/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.